src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -3909,6 +3909,8 @@ private static RelocType GetRelocType(TargetArchitecture targetArchitecture, ush
                     const ushort IMAGE_REL_ARM64_BRANCH26 = 3;
                     const ushort IMAGE_REL_ARM64_PAGEBASE_REL21 = 4;
                     const ushort IMAGE_REL_ARM64_PAGEOFFSET_12A = 6;
+                    const ushort IMAGE_REL_ARM64_SECREL_HIGH12A = 0xA;
+                    const ushort IMAGE_REL_ARM64_SECREL_LOW12L = 0xB;
```

Comment:
Nit: some constants here match values from `RelocType` and some match the actual platform-specific values (for the ease of disassembling?).

If that is a case, then maybe always use platform-specific values? 
I.E. `IMAGE_REL_ARM64_TLSDESC_CALL` would match `R_AARCH64_TLSDESC_CALL`, which is 0x239`

Or the other way - just use `RelocType` values for all?

Reply:
Does not need to be addressed in this change since the inconsistency is already there.

Reply:
yeah, probably will not include in this PR.